.github/workflows/ci.yml

```diff
@@ -334,6 +444,62 @@ jobs:
         fi
         cd ../../
 
+    - name: Build LLVM/Cling on Windows systems if the cache is invalid
+      if: ${{ runner.os == 'windows' && steps.cache.outputs.cache-hit != 'true' }}
+      run: |
+
+        if ( "${{ matrix.cling }}" -imatch "On" )
+        {
+          git clone https://github.com/root-project/cling.git
+          cd ./cling 
+          git checkout tags/v${{ matrix.cling-version }}
+          cd ..
+          git clone --depth=1 -b cling-llvm${{ matrix.clang-runtime }} https://github.com/root-project/llvm-project.git
+        }
+        else
+        {
+          git clone --depth=1 -b release/${{ matrix.clang-runtime }}.x https://github.com/llvm/llvm-project.git
+        }
+        cd llvm-project
+        # Apply patches
+        echo "Apply clang${{ matrix.clang-runtime }}-*.patch patches:"
+        compgen -G "../patches/llvm/clang${{ matrix.clang-runtime }}-*.patch" > /dev/null && find ../patches/llvm/clang${{ matrix.clang-runtime }}-*.patch -exec basename {} \; -print -exec git apply {} \;
```

Comment:
@vgvassilev this PR is expected to currently fail to build on Windows at the moment, as I don't know what the windows equivalent of this linux/osx command is. Any ideas?

Reply:
This has been solved now